### ISSUE TITLE
Error 500, hubDatabase in not defined

### ISSUE THREAD

**Issue by paeltech** [nuxt-hub/core#165]
**Describe the bug**
I have managed to deploy my app on nuxtHub and I can see my database data on DEvTools but I can't query the data on my local app getting error "500 hubDatabase is not defined"

**Steps to reproduce**
1. Create a NuxtHub project
2. Deploy to NuxtHub
3. Add access to remote data on localhost
4. Run npm install
5. Declare const = hubDatabase()
6. Get the error

**Expected behavior**
Expected to be able to access the hubDatabase() server composable

**Comment by ra-jeev** [comment]
By any chance related to this issue #159?

**Comment by Atinux** [comment]
Can you confirm you have in your `nuxt.config.ts`:

```ts
hub: {
  database: true
}
```

**Comment by maclong9** [comment]
I'm getting the same error, and I definitely have the below in my `nuxt.config.ts`

```
hub: {
  database: true,
}
```

**Comment by ra-jeev** [comment]
> I'm getting the same error, and I definitely have the below in my `nuxt.config.ts`
> 
> 
> 
> ```
> 
> hub: {
> 
>   database: true,
> 
> }
> 
> ```

Are you running locally, or trying to access the remote database. Providing the steps that you followed would help.

**Comment by ra-jeev** [comment]
So you're adding it to an existing project. Just to confirm, is the project already deployed on Cloudflare? (Either through NuxtHub or manually)?

**Comment by maclong9** [comment]
The project is deployed via NuxtHub.

**Comment by ra-jeev** [comment]
I just now tried with a fresh test project with basic Nuxt template (`npx nuxi init`). Steps:

1. Added NuxtHub `npx nuxi module add hub`
2. Added wrangler `npm install --save-dev wrangler`
3. Create a simple API route with hubDatabase method call. And invoked this route from frontend
4. Deployed the project using CLI `npx nuxthub deploy` 

And it seems to work as expected. To test out an error scenario, I did not create any table on the remote, yet tried inserting data to it, and I do get a `500 Error` but that is expected.

`POST 500 /api/_hub/database/run
[nuxt] [request error] [unhandled] [500],D1_ERROR: no such table: time: SQLITE_ERROR`

I would suggest that you create a fresh issue with all the details related to your project.
